src/components/templates/Auditorium/components/Section/Section.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useMemo, useCallback } from "react";
+import { useHistory } from "react-router-dom";
+import { useParams } from "react-router";
+
+import { AuditoriumVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+import { enterVenue } from "utils/url";
+
+import {
+  useAuditoriumSection,
+  useAuditoriumGrid,
+} from "hooks/auditoriumSections";
+
+import { BackButton } from "components/atoms/BackButton";
+import { IFrame } from "components/atoms/IFrame";
+
+import "./Section.scss";
+
+// If you change this, make sure to also change it in Section.scss
+export const SECTION_SEAT_SIZE = "var(--section-seat-size)";
+export const SECTION_SEAT_SIZE_MIN = "var(--section-seat-size-min)";
+export const SECTION_SEAT_SPACING = "var(--section-seat-spacing)";
+
+export interface SectionProps {
+  venue: WithId<AuditoriumVenue>;
+}
+
+export const Section: React.FC<SectionProps> = ({ venue }) => {
+  const {
+    iframeUrl,
+    rows: venueRowsCount,
+    columns: venueColumnsCount,
```

Comment:
Not sure why you are saying this. There is no such thing in the Auditorium venue type 
![image](https://user-images.githubusercontent.com/63194656/124765540-bd30f200-df3e-11eb-85d1-6e66a388b635.png)

Reply:
@mike-lvov  we don't, but looks like it's because the properties weren't added to the right interface.
`auditoriumRows` and `auditoriumColumns` are in `BaseVenue` (and `AuditoriumVenue extends BaseVenue`).

I suggested that because I thought that in this case it'd be easier to switch to Auditorium V2 (if we would use the same venue properties for the same purposes).

Reply:
@sunny-viktoryia you are right. Changed in https://github.com/sparkletown/sparkle/pull/1251/commits/5bbf188eadabe69e3ca5046a8b4e3a8d09d1535e